staff/bernat-casasus/restaurants-app.1/scripts/web-components-1.0.0.js

```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+/**
+ * Web Components lib
+ * 
+ * @author manuelbarzi
```

Comment:
qui es l'autor?

Reply:
Please If posible use english to send comments!